yellowbrick/features/pca.py

```diff
@@ -121,6 +127,7 @@ def __init__(self,
 
         # Visual Parameters
         self.color = color
+        self.legend_labels = legend_labels
```

Comment:
Let's change to `self.classes_ = classes`. Since this visualizer inherits from `MultiFeatureVisualizer` rather than `DataVisualizer`, we'll need to explicitly set this (whereas with a `DataVisualizer`, `self.classes_` would be automatically created if the user passed in something for the `classes` param)